Mathlib/Topology/LocallyConstant/Basic.lean

```diff
@@ -554,10 +554,11 @@ end Indicator
 
 section Equiv
 
-/-- The equivalence between `LocallyConstant X Z` and `LocallyConstant Y Z` given a
-    homeomorphism `X ≃ₜ Y` -/
+/--
+The equivalence between `LocallyConstant X Z` and `LocallyConstant Y Z` given a
+homeomorphism `X ≃ₜ Y`
+-/
 @[simps]
-noncomputable
```

Comment:
Is there a particular reason you are removing all those `noncomputable`? I suspect they were there for performance reasons.

Reply:
They were there because `LocallyConstant.comap` was noncomputable before I refactored it to take an argument of the form `C(-,-)`. I'm pretty sure I added all the declarations that I'm removing `noncomputable` from, and I know I wasn't thinking about performance then. 

There may very well be performance reasons to keep them though. What do you think?

Reply:
Okay, if you added the `noncomputable` then no problem!